fvm/evm/types/address.go

```diff
@@ -57,6 +58,15 @@ func (fa Address) Bytes() []byte {
 	return fa[:]
 }
 
+// String returns the hex encoding of the address
+// it returns empty string if address is empty
+func (fa Address) String() string {
+	if fa == EmptyAddress {
+		return ""
+	}
+	return hex.EncodeToString(fa[:])
```

Comment:
Should we maybe change this to:
```go
return fa.ToCommon().Hex()
```
?
The difference being that the latter returns the check-summed variant, e.g.:
from: **99466ed2e37b892a2ee3e9cd55a98b68f5735db2**
to: **0x99466ED2E37B892A2Ee3E9CD55a98b68f5735db2**

Reply:
Note: I also made a change in https://github.com/onflow/flow-go/pull/5416, which I guess is conflicting.

Reply:
good idea, I'll use your changes, but I think the issue is still the empty address, it would result in growth in all events.

Reply:
now that I'm looking into it, the checksum is a post-process only input-dependent, not sure if it makes sense to do it on-chain and not off-chain later?